easybuild/easyconfigs/a/Amber/Amber-16-iomkl-2016.07-AmberTools-16-patchlevel-5-14-serial.eb

```diff
@@ -0,0 +1,32 @@
+#
+# Based on https://raw.githubusercontent.com/hpcugent/easybuild-easyconfigs/master/easybuild/easyconfigs/a/Amber/Amber-14-intel-2016a-AmberTools-15-patchlevel-13-13.eb
+#
+name = 'Amber'
+version = '16'
+ambertools_ver = '16'
```

Comment:
just use `version` or `%(version)s`? why introduce another variable for the same thing?